src/Components/Individual-Review.js

```diff
@@ -0,0 +1,90 @@
+import styles from "../Stylesheets/Individual-Review.module.css";
+import { useState, useEffect } from "react";
+import { getIndividualReview } from "../Utils/Api-Calls";
+import { useParams } from "react-router-dom";
+import timeConv from "../Utils/Time-Conversion";
+import GridLoader from "react-spinners/GridLoader";
+
+const Review = () => {
+  const [review, setReview] = useState({
+    created_at: "2021-01-18T10:01:41.251Z",
```

Comment:
If it wasn't there it was generating an error on the initial render because at some point in the conversion function it does a trim and errors when the string is empty. Upon typing this, I realise I could also solve this with a conditional within the function